app/services/folio_graphql_client.rb

```diff
@@ -152,6 +154,31 @@ def instance(hrid:)
     data&.dig('data', 'instances', 0)
   end
 
+  def due_date(instance_id:)
+    data = post_json('/', json:
+      {
+        query:
+          <<~GQL
+            query DueDate {
+              availability(id: "#{instance_id}") {
```

Comment:
Is RTAC fast enough that we can just do this in the original query?

Reply:
It's still adds significant slowness, especially in the worst cases.